stories/atoms/components/list/List.stories.mdx

```diff
@@ -0,0 +1,284 @@
+        type: { summary: 'JSX.Element' }
+      }
+    },
+    rightIcon: {
+      control: false,
+      table: {
+        type: { summary: 'JSX.Element' }
+      }
+    }
+  }}
+  args={{
+    items: [
+      {
+        label: 'Item 1',
+        description: 'Description 1'
+      },
+      {
+        label: 'Item 2',
+        description: 'Description 2'
+      },
+      {
+        label: 'Item 3',
+        description: 'Description 3'
+      },
+      {
+        label: 'Item 4',
+        description: 'Description 4'
+      },
+      {
+        label: 'Item 5',
+        description: 'Description 5'
+      }
+    ],
+    leftIcon: <Icon name="wallet" />,
+    rightIcon: <Icon name="close" />
+  }}
+>
+  {args => <List {...args} />}
+</Story>
+
+## Properties
+
+<ArgsTable story="Overview" />
+
+## ListItem
+
+The property `items` provides the list of items to display.
+
+<Story
+  name="ListItem"
+  argTypes={{
+    description: {
+      table: {
+        type: { summary: 'string | JSX.Element' }
+      }
+    },
+    leftIcon: {
+      control: false,
+      table: {
+        type: { summary: 'JSX.Element' }
+      }
+    },
+    rightIcon: {
+      control: false,
+      table: {
+        type: { summary: 'JSX.Element' }
+      }
+    },
+    items: {
+      table: {
+        disable: true
+      }
+    }
+  }}
+  args={{
+    label: 'Item',
+    description: 'Description',
+    leftIcon: <Icon name="wallet" />,
+    rightIcon: <Icon name="close" />
+  }}
+>
+  {args => <ListItem {...args} />}
+</Story>
+
+An element of the `List` is a `ListItem` component.
+
+### Properties
+
+<ArgsTable of={ListItem} />
+
+### Label
+
+The mandatory property `label` allows you to display a label which is the main title of the item.
+
+<Canvas>
+  <Story name="Name">{() => <ListItem label="The label of the item." />}</Story>
+</Canvas>
+
+### Description
+
+The `description` property allows you to provide additional information. It can be either a `string` or a `JSX.Element`.
+
+<Canvas>
+  <Story name="Description">
+    {() => (
+      <List
+        items={[
+          {
+            label: 'Item1',
+            description: 'The additional information of the Item 1'
+          },
+          {
+            label: 'Item2',
+            description: (
+              <div>
+                <Typography as="div" color="text" fontSize="small">
+                  Additional information in Item 2 that may be more complex than a simple `string`.
+                  The description can display any content.
+                </Typography>
+                <ProgressBar label="okp4.pdf" minValue={0} maxValue={195} currentValue={23} />
+              </div>
+            )
+          }
+        ]}
+      />
+    )}
+  </Story>
+</Canvas>
+
+## Icons
+
+The `leftIcon` and `rightIcon` properties allow you to provide visual information for each item in the list.  
+Theses properties override the `leftIcon` and `rightIcon` properties of the `List` if they are provided.
```

Comment:
This section is quite confusing, related to my comments in components on icon properties